### ISSUE TITLE
Idea For a Project

### ISSUE THREAD

**Issue by DHRUV-SHERE** [SwastikSadhna/hinduvidhyapith#9]
I have a new idea which I got from vinay's portfolio where he has used 3d type model so can we make a page where the user can visit the most famous hindu temple available on Indian map by using the controls where he can get details about that temple

**Comment by Neesargsoni** [comment]
Yeah, we can provide a 3d view of temple and also add some description whenever user clicks on any part of temple then that part would display some details about the temple.
